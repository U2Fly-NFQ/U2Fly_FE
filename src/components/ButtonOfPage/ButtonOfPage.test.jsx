import renderer from 'react-test-renderer'
import React from 'react'
import AddminDiscount from './'

describe('Test snapshot', () => {
  it('Should match snapshot', () => {
    const tree = renderer.create(<AddminDiscount />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
