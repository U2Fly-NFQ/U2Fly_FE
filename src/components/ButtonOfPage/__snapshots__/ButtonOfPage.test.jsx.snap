// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test snapshot Should match snapshot 1`] = `
<div
  className="form-submit"
>
  <button
    className="btn btn-primary btn-md"
    id=""
    type="submit"
  />
</div>
`;
